pkg/controller/issueapi/issue.go

```diff
@@ -296,13 +298,9 @@ func (c *Controller) getAuthorizationFromContext(r *http.Request) (*database.Aut
 	return authorizedApp, currentUser, nil
 }
 
-func (c *Controller) recordCapacity(ctx context.Context, realm *database.Realm, remaining uint64) {
-	if !realm.AbusePreventionEnabled {
-		return
-	}
+func (c *Controller) recordCapacity(ctx context.Context, limit, remaining uint64) {
```

Comment:
I changed this to rely on the value that comes from the limiter, not the value on the realm. The enforcement happens based on what's in the limiter, so we should alert and record off of that.